website/content/docs/ui/custom-messages.mdx

```diff
@@ -0,0 +1,151 @@
+   of the current session.
+
+1. **Messages are intrusive**. Limit the number of active messages to minimize
+   the intrusion on users and reduce the chances that they will dismiss the
+   message without reading it.
+
+1. **Messages are inheritable**. Child namespaces inherit all messages created
+   on the parent namespace. Take advantage of inheritance to reach the greatest
+   number of users with the smallest number of messages.
+
+1. **Delete old messages**. Vault supports a maximum of 100 messages per namespace at a
+   time. Practice good message hygiene by regularly deleting expired and outdated
+   messages.
+
+</Tip>
+
+## Before you start
+
+- **You must have Vault Enterprise 1.16.0 or higher installed.**
+- **You must have the appropriate permissions**:
+  - You must have `list` permission for the `sys/config/ui/custom-messages` endpoint.
+  - To **create messages**, you must have `read` permission for the `sys/config/ui/custom-messages/:id` endpoint and `create` permission for the `sys/config/ui/custom-messages` endpoint.
+  - To **edit messages**, you must have `read` and `update` permission for the `sys/config/ui/custom-messages/:id` endpoint.
+  - To **delete messages**, `delete` permission for the `sys/config/ui/custom-messages/:id` endpoint.
+
+## Add a custom message
+
+1. Navigate to the **Settings** section in the Vault UI sidebar and select **Custom
+   Messages**.
+
+1. On the **Custom messages** page, select whether you want the message
+   to appear on the Vault UI login page or after a user logs in.
+
+1. Select the **+ Create message** button in the toolbar to open the **Create message** form.
+
+1. On the **Create message** form:
+
+   - Select the locations where your message should appear.
+   - Select the message type.
+   - Provide a title and the message text. For important messages, we recommend
+     keeping the text short and including a link to more information rather
+     than writing a longer message that users may not take the time to read.
+   - Set a start time when your message will publish. By default, messages
+     publish at midnight in your local timezone.
+   - Set an optional end time for the message. By default, messages do not
+     expire.
+
+1. Click **Preview** to see how your message will appear to users.
+
+1. Click **Create message** to save your new message.
+
+## Create messages for a specific namespace
+
+Child [namespaces](/vault/docs/enterprise/namespaces) inherit all messages
+created on the parent namespace. For example, if your cluster has the following
+namespace hierarchy:
+
+<CodeBlockConfig hideClipboard>
+
+```plaintext
+─ admin
+    ├── finance
+    └── marketing
+        ├── digital-marketing
+        └── events
+```
+
+</CodeBlockConfig>
+
+Custom messages created on the `admin` namespace apply to all child namespaces:
+`fiance`, `marketing`, `marketing/digital-marketing` and `marketing/events`.
```

Comment:
Based on Finn's feedback from the other PR, there is a spelling error `finance`.